vividus-plugin-web-app/src/main/resources/properties/profile/web/docker/profile.properties

```diff
@@ -0,0 +1,3 @@
+selenium.grid.enabled=true
+selenium.grid.capabilities.platformName=linux
+selenium.grid.capabilities.browserName=${selenium.browser}
```

Comment:
we need to document this profile

Reply:
https://github.com/vividus-framework/vividus/pull/2773/files#diff-5f5cfd809148588b4ac9c13f1af82fc8aa5c50b0f862c76e7f6bb14cd00b74d1R210-R222